networkx/algorithms/tests/test_min_edge_cover.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+from nose.tools import *
+import networkx as nx
+
+
+def test_trivial1():
```

Comment:
`test_trivial` is a bad name for a test. What are you actually testing? That the empty graph has an empty edge cover. So call it `test_empty_graph` or something like that. Same for the rest.